frontend/src/constants/Routes.ts

```diff
@@ -13,3 +15,9 @@ export const CREATE_ENTITY_PAGE = "/entity/create";
 export const UPDATE_ENTITY_PAGE = "/entity/update";
 
 export const HOOKS_PAGE = "/hooks";
+
+export const MAGAZINE_REVIEW_PAGE = "/users/magazine-review";
```

Comment:
For these route names I think MAGAZINE_REVIEWS_PAGE="/reviews" makes more sense? But not sure whether /users/reviews works better since it is technically blocked by authen
And for ADMIN_DASHBOARD_PAGE = "/users/admin/dashboard" should this just be "/dashboard" maybe? I'm not too familiar with best practices for react routes tbh. @uwblueprint/ccbc do you guys have any suggestions?